src/App.js

```diff
@@ -171,6 +173,17 @@ class OldApp extends Component {
      * Needs to be called AFTER FCM token is loaded
      */
     initWalletConnectListeners();
+
+    /**
+     * Launch the review prompt 30 seconds after the app is launched
+     * This is to avoid the review prompt showing up when the app is
+     * launched and not shown yet.
+     */
+    InteractionManager.runAfterInteractions(() => {
+      setTimeout(() => {
+        handleReviewPromptAction(ReviewPromptAction.TimesLaunchedSinceInstall);
+      }, 30_000);
+    });
```

Comment:
@skylarbarrera is there a way to make sure this doesn't interrupt the user's workflow? I want to be mindful of that here especially.

Reply:
we'd have to tie it to some stateful listener that could cancel/delay the timeout

Reply:
yeah I think this is fine for now imo, if anything make it less time so they can't get as far into something, I think 10-15 is a happy medium